Added max_string_lengthj option (#7465)
getsentry/sentry-docs: src/platforms/common/configuration/options.mdx

</ConfigKey>

<ConfigKey name="max-value-length" supported={["php", "python"]}>

The number of characters after which the values containing text in the event payload will be truncated (defaults to `1024`).

> **Note:** If you set this to a very big value, it can happen that the event as a whole becomes bigger that 1 MiB. When this threshold is reached the event will be dropped by Sentry.

Reviewer: ```suggestion
<Note>
 If the value you set for this is exceptionally large, the event may exceed 1 MiB and will be dropped by Sentry.
 </Note>
```
Is it possible to specify what "exceptionally large" or "very big" is?
